server/subsonic/responses/responses.go

```diff
@@ -281,6 +281,7 @@ type User struct {
 	JukeboxRole         bool   `xml:"jukeboxRole,attr"            json:"jukeboxRole"`
 	ShareRole           bool   `xml:"shareRole,attr"              json:"shareRole"`
 	VideoConversionRole bool   `xml:"videoConversionRole,attr"    json:"videoConversionRole"`
+	SyncPlaylist        bool   `xml:"syncPlaylist,attr"           json:"syncPlaylist"`
```

Comment:
correct, this is not part of the subsonic API, but as for if it is strictly necessary, if we want to allow the user to log in from a device that has no cache and we want that new device to respec the sync settings we are required to store it somewhere, i have put it to the subsonic since it is related to it, but since it is not part of the API it can be  moved, but for my testing it didn't affect other platforms